src/cacheServerClient/ICacheServerClient.ts

```diff
@@ -23,11 +23,11 @@ export interface ICacheServerClient {
     getAppDefinition: ({ namespace }: Pick<ClaimsQueryParams, "namespace">) => Promise<IAppDefinition>;
     getApplicationRoles: ({ namespace }: Pick<ClaimsQueryParams, "namespace">) => Promise<IRole[]>;
     getOrganizationRoles: ({ namespace }: Pick<ClaimsQueryParams, "namespace">) => Promise<IRole[]>;
-    getOrganizationsByOwner: ({
-        owner,
-        excludeSubOrgs,
-    }: Pick<ClaimsQueryParams, "owner" | "excludeSubOrgs">) => Promise<IOrganization[]>;
-    getApplicationsByOwner: ({ owner }: Pick<ClaimsQueryParams, "owner">) => Promise<IApp[]>;
+    getOrganizationsByOwner: (
+        owner: ClaimsQueryParams["owner"],
+        opts?: { withRelations?: boolean },
```

Comment:
I think it would be better to create an interface with possible options. Current approach can lead to the same issue that we have fixed here https://github.com/energywebfoundation/iam-client-lib/pull/279

Reply:
Named interface would prevent name confusion if it was used in both libraries. This `opts` object is used only in two functions here. I think it would be an overkill to define such simple type in one of the lib and export it into other